webhooks/webhooks.constant.ts

```diff
@@ -0,0 +1,91 @@
+export class WebhooksConstant {
+    static readonly webhooksUrl = '/f/page?ixPlugin=21&pg=pgPluginConfig';
+
+    static readonly attributes = Object.freeze({
+        text: Object.freeze({
+            header: 'Webhooks',
+            addNewHook: 'Add New Hook',
+            editHook: 'Editing Hook',
+            deleteHook: 'Delete Hook',
+            ok: 'o',
```

Comment:
Codacy has a fix for the issue: [Strings must use doublequote.](https://app.codacy.com/gh/davidgunadi/hello-world/pullRequest?prid=8400130)
```suggestion
            ok: "o",
```